src/views/accounts/AccountChangeSecret/ExecutionDetail/AccountChangeSecretRecord.vue

```diff
@@ -56,7 +56,7 @@ export default {
               can: this.$hasPerm('accounts.view_account'),
               getTitle: ({ row }) => row.account.username,
               getDrawerTitle: ({ row }) => row.account.username,
-              getRoute({ row }) {
+              getRoute: ({ row }) => {
                 return {
                   name: 'AssetAccountDetail',
                   params: { id: row.account.id }
```

Comment:
It seems that there is no difference between the code you posted and others. Both codes seem to be perfectly valid and do not contain any irregularities, potential issues, or optimization suggestions related to knowledge cutoff of September 1, 2021.

The only change made since was renaming `GenericListTable` import (`actionsFormatter`) instead of using it (it's just an alias), which I consider negligible if it improves readability in some context. Otherwise, both versions seem to work well as intended.

Here we go:

No major changes were done here.

Please let me know how this works based on your requirements!